0-ad-history-mod-package/simulation/templates/structures/han/civil_centre.xml

```diff
@@ -10,6 +10,12 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Guān Shǔ</SpecificName>
+    <History>
+         The Han empire was absolutely governed by an emperor who would pass on the throne to his eldest son. He was the supreme lawgiver, chief commander of the military, and designator of senior government officials. However, the emperor was assisted by a complex government system. It consisted of numerous layers and institutions and was meant to simplify political procedures without letting anyone (except the emperor) amass power. The system was structured as follows (simplified):
```

Comment:
The first paragraph is contently and structurally good, however some of the way words were used isn't standard for En_US or I there are better words. In my suggestion, I cut the "designator of senior officials"  phrase need its sentence because it's awkward, and I couldn't figure out a better way to reword it, as well as I thought that bit of the government would be obvious.

Finally, the () with obviously in it, I added because it seemed to work better with absolutely. However, this adds a bit of humor to the article... your call if you want it. 
Here's my suggested change: Feel free to find your own better ways of saying things
```suggestion
         The Han empire was (obviously) governed by an emperor. One (obviously) descended from the Han dynasty. He was the supreme authority in both legal and military matters. However, the emperor was assisted by a complex bureaucracy. This bureaucracy had many layers and institutions which were supposed to simplify political procedures without letting anyone (except the emperor) amass power. The system was structured as follows ( obviously simplified):
```